src/shared/sam/debugger/goSamDebug.ts

```diff
@@ -165,13 +165,25 @@ async function makeInstallScript(debuggerPath: string, isWindows: boolean): Prom
     const delvePath: string = path.posix.join(debuggerPath, 'dlv')
     const installOptions: SpawnOptions = { env: { ...process.env } }
     let delveVersion: string = ''
+    installOptions.env!['GO111MODULE'] = 'off'
```

Comment:
Can you add a comment here why this works?

Reply:
With Go1.16 `GO111MODULE` is on by default. This causes Go to require the existence of a `go.mod` file or else it gives up. Setting the environment variable prevents Go from trying to find the manifest file and uses the `GOPATH` provided later on. Not sure why Go couldn't find a manifest file on Justin's machine, though now it doesn't matter with this patch.